integration/main_test.go

```diff
@@ -29,20 +29,18 @@ import (
 	"time"
 
 	"github.com/containerd/containerd"
+	"github.com/containerd/cri/integration/remote"
```

Comment:
lot of code to move and "support" for one integration test... hmmm

Reply:
perhaps we don't need this integration test if it's already in a e2e node bucket ?